packages/shared/friends/sagas.ts

```diff
@@ -1370,10 +1384,14 @@ function* handleJoinOrCreateChannel(action: JoinOrCreateChannel) {
     }
 
     // get or create channel
-    const { created, conversation } = yield apply(client, client.getOrCreateChannel, [channelId, []])
+    const { created, conversation }: { created: boolean; conversation: Conversation } = yield apply(
+      client,
+      client.getOrCreateChannel,
+      [channelId, []]
+    )
 
     const channel: ChannelInfoPayload = {
-      name: action.payload.channelId,
+      name: conversation.name ?? action.payload.channelId,
```

Comment:
Should we have the `channelId` by default or remove it completely and ensure that `conversation.name` is present?

Reply:
would need to rework the `matrix-client` API and ensure somehow that the field will always be present, not sure right now if that's possible